readthedocs/embed/views.py

```diff
@@ -89,23 +89,21 @@ class EmbedAPIBase(APIView):
 
     ### Arguments
 
-    We support two different ways to query this API:
+    We support two different ways to query the API:
 
-        * project (required)
-        * version (required)
-        * doc
-        * section
-        * path
+    * project (required)
+    * version (required)
+    * doc or path (required)
+    * section
 
     or:
 
-        * url (required)
+    * url (with fragment) (required)
 
     ### Example
 
-        GET https://readthedocs.org/api/v2/embed/?project=requests&doc=index&section=User%20Guide&path=/index.html  # noqa
-
-        GET https://readthedocs.org/api/v2/embed/?url=https://docs.readthedocs.io/en/latest/features.html%23github-bitbucket-and-gitlab-integration # noqa
+    - GET https://readthedocs.org/api/v2/embed/?project=requestsF&version=latest&doc=index&section=User%20Guide&path=/index.html  # noqa
+    - GET https://readthedocs.org/api/v2/embed/?url=https://docs.readthedocs.io/en/latest/features.html%23github-bitbucket-and-gitlab-integration # noqa
```

Comment:
Does this #noqa show up in the docs? We should figure out a better way to ignore that, if so.

Reply:
Yeah it does, I was able to disable the linter here with pylint ignore and #noqa